include/fastrtps/rtps/common/SequenceNumber.h

```diff
@@ -84,22 +93,18 @@ struct RTPS_DllAPI SequenceNumber_t
 
 
     //! Increase SequenceNumber in 1.
-    SequenceNumber_t& operator++() noexcept
+    SequenceNumber_t& operator ++() noexcept
     {
-        if(low == UINT32_MAX)
```

Comment:
Can the corresponding header be removed?

Reply:
done!